### PR TITLE
Simplified/Reduced context property names.

### LINKED ISSUE #323: Context attribute naming consistency
In last weeks call a comment was made (Tim from AWS?) relating to the potential for renaming some of the context attributes; i thought it would make sense to raise it for discussion here prior to opening what would be quite a large PR.

The proposal (as i understand it) would be to rename along these lines:

| Current | Proposed |
| --- | --- |
| eventType | type |
| cloudEventsVersion | version |
| eventID | id |
| eventTime | time |

An alternative argument might be to ensure all context attributes follow a consistent naming structure, eg :

| Current | Proposed |
| --- | --- |
| eventType | eventType |
| source | eventSource |
| data | eventData |

### PR DESCRIPTION
**Pull request by JemDay**
Simplified/Reduced context property names.

Closes #323

### CODE REVIEW

File: community/contributors.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ Contributions do not constitute an official endorsement.
   * Chad Arimura - [@carimura](https://github.com/banisadr)
   * Stanley Halka - [@shalka](https://github.com/banisadr)
   * Travis Reeder - [@treeder](https://github.com/banisadr)
+* **PayPal**
+  * Vladimir Bacvanski
+  * Jem Day - [@JemDay](https://github.com/JemDay)
```
Comment (duglin) [inline community/contributors.md]: EVERYONE - if you've submitted an issue or PR please feel free to open a PR to add your name to this doc.

File: spec.md [hunk 2]
```diff
@@ -273,7 +273,7 @@ encapsulated within the `data` attribute.
 
 ### data
 * Type: `Any`
-* Description: The event payload. The payload depends on the eventtype and
+* Description: The event payload. The payload depends on the `type` and
   the schemaURL. It is encoded into a media format
```
Comment (duglin) [inline spec.md]: if you use back-ticks on "type" then for consistency you should on "schemaURL" on the next line, and "contenttype" - and we should be consistent on the case: e.g. schemaURL vs contenttype

**Comment by duglin** [comment]
@JemDay I updated the title of the PR (to be more descriptive) and added the "Closes #323" so it'll automagically close the issue. Hope that's ok.

File: spec.md (hunk 2 above)
Comment (JemDay) [reply spec.md]: Done

**Comment by rperelma** [comment]
Everything LGTM, except for one thing: I would strongly prefer to keep `eventid` instead of `id`

**Comment by JemDay** [comment]
> Everything LGTM, except for one thing: I would strongly prefer to keep `eventid` instead of `id`

NOTED - I think this goes against the _consistency_ theme slightly but is probably a good subject for discussion at the next session (@duglin)

**Comment by duglin** [comment]
yup - I've love to hear what others think, especially since I'm hoping we can resolve that one issue during the call and approve it. So if you can't make the call and have strong opinions on this point please add them here as comments.

**Comment by clemensv** [comment]
LGTM

**Comment by duglin** [comment]
@JemDay can you rebase? or do you want me to try it?

**Comment by JemDay** [comment]
> @JemDay can you rebase? or do you want me to try it?

Let me try ....

**Comment by duglin** [comment]
Approved on the Nov 8 call

**Pull request merged**